cmake/USDPluginTools.cmake

```diff
@@ -417,11 +417,13 @@ function(_usd_cpp_library NAME)
     else()
         # Setup SOVERSION & VERSION properties to create
         # NAMELINK, SONAME, and actual library with full version suffix.
-        set_target_properties(${NAME}
-            PROPERTIES
-                SOVERSION ${CMAKE_PROJECT_VERSION_MAJOR}
-                VERSION ${CMAKE_PROJECT_VERSION}
-        )
+        if (!APPLE)
```

Comment:
Hey thanks for this PR - I'm not familiar with OSX development enough to comment whether or not this is the best approach... will ask around and read up over the weekend and follow up on this next week.